### PR TITLE
Add the first test for dynamic validator (N->N')

### PR DESCRIPTION
**Pull request by majecty**
This PR depends on #1677.

### CODE REVIEW

File: test/src/e2e.dynval/dv.n'.test.ts [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+            for (let i = 0; i < otherDynNodes.length; i++) {
+                await otherDynNodes[i].waitForTx(changeTx);
+            }
+
+            const margin = 1.1;
+
+            await waitForTermChange(1, TERM_SECONDS * margin);
+
+            {
+                const authors = await otherDynNodes[0].sdk.rpc.sendRpcRequest(
```
Comment (foriequal0) [inline test/src/e2e.dynval/dv.n'.test.ts]: `const authors = await otherDynNodes[0].sdk.rpc.chain.getPossibleAuthors();`

Comment (majecty) [reply test/src/e2e.dynval/dv.n'.test.ts]: `getPossibleAuthors` not in the current SDK version.

Comment (foriequal0) [reply test/src/e2e.dynval/dv.n'.test.ts]: How about using `stake.getPossibleAuthors`?

Comment (majecty) [reply test/src/e2e.dynval/dv.n'.test.ts]: I updated!.

**Pull request merged**